vuepress/docs/next/docs/reference/caching-and-clustering.md

```diff
@@ -6,13 +6,13 @@ The Entando App Engine can be deployed as a clustered set of instances using the
 
 This guide demonstrates how to configure and test a clustered instance of the Entando App Engine.
 
-Microservices clustering that adds functionality to an Entando Application is different from clustering used by the Entando App Engine. Microservices rely on a custom clustering configuration and setup based on implementation and selections made during their creation. Refer to documentation addressing clustered microservices and caching implementation for configuration and deployment details.
+Microservices clustering that adds functionality to an Entando Application is different from clustering used by the Entando App Engine. Microservices rely on a custom clustering configuration and setup based on implementation and selections made during their creation. Refer to documentation addressing [clustered microservices and caching implementation](../../tutorials/devops/caching-and-clustering.md) for configuration and deployment details.
 
 ## Storage Requirements for Clustered Entando Applications
 
-To scale an Entando Application across multiple nodes a storage class that supports a `ReadWriteMany` access policy must be instantiated. There are a number of ways to accomplish this, inclusive of dedicated storage providers such as GlusterFS. Cloud Kubernetes providers also offer clustered storage options specific to their implementation, such as Google Cloud File in GKE or Azure Files in AKS.
+To scale an Entando Application across multiple nodes, a storage class that supports a `ReadWriteMany` access policy must be initiated. There are a number of ways to accomplish this, inclusive of dedicated storage providers such as GlusterFS. Cloud Kubernetes providers also offer clustered storage options specific to their implementation, such as Google Cloud File in GKE or Azure Files in AKS.
```

Comment:
'initiated' is usually used to describe a process that starts and then runs. In this case 'instantiated' or 'created' or even 'set up' would be clearer.